onedal/linear_model/logistic_regression.cpp

```diff
@@ -95,8 +95,14 @@ auto get_onedal_result_options(const py::dict& params) {
             else if (match.str() == "iterations_count") {
                 onedal_options = onedal_options | result_options::iterations_count;
             }
-            else
+#if ONEDAL_VERSION >= 20240400
+            else if (match.str() == "inner_iterations_count") {
+                onedal_options = onedal_options | result_options::inner_iterations_count;
+            }
+#endif 
```

Comment:
No way to test it until oneDAL version is updated to 2024.4

Reply:
yes, so I suggest merging https://github.com/oneapi-src/oneDAL/pull/2671

Reply:
UPD: I changed required onedal version to 2024.3, since it seems that this functionality will be available in earlier version